libs/zjobs/src/zjobs.zig

```diff
@@ -577,8 +563,15 @@ pub fn JobQueue(
             const prereq_size = @sizeOf(JobId);
             const max_prereqs = (max_job_size - jobs_size) / prereq_size;
 
-            jobs: *Self = .{},
```

Comment:
I'm surprised this would even compile?  Probably needed to be `undefined`?

Reply:
Can try this when i fix the other issues and try without init again 👍